src/frequenz/client/common/pagination/__init__.py

```diff
@@ -0,0 +1,25 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Common code and utilities for Frequenz API clients.
+
+TODO(cookiecutter): Add a more descriptive module description.
+"""
```

Comment:
```suggestion
"""Common types to handle pagination of request results."""
```
Or something more along those lines?